astroquery/linelists/cdms/core.py

```diff
@@ -0,0 +1,313 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+import os
+import warnings
+
+from bs4 import BeautifulSoup
+import astropy.units as u
+from astropy.io import ascii
+from astroquery.query import BaseQuery
+from astroquery.utils import async_to_sync
+# import configurable items declared in __init__.py
+from astroquery.linelists.cdms import conf
+from astroquery.linelists.jplspec import lookup_table
```

Comment:
This module is not yet moved, so keep the original import
```suggestion
from astroquery.jplspec import lookup_table
```

Reply:
I... didn't make that change.  Huh.